apps/android/boot.js

```diff
@@ -251,8 +251,17 @@
   // Battery monitor
   function sendBattery() { gbSend({ t: "status", bat: E.getBattery(), chg: Bangle.isCharging()?1:0 }); }
   Bangle.on("charging", sendBattery);
+  function sendEnv() {
+    gbSend({
```

Comment:
Do we want to keep this in one line? My understanding is that we minify the js for "release", so it should only become slightly slower during development.

Reply:
I'm not quite sure I understand the question? But it'd be better to put it straight in the `connect` handler - not for speed but because I don't believe the minified will inline it, and then it'll leave the extra function stored in RAM. Not a big deal on Bangle.js 2, but on v1 we're pretty tight ram-wise.

sendBattery is only a function because we're calling it from more than one place